### PR TITLE
upgrade to use ubuntu 16.05 and modify structure to allow changing without rebuilding

### PR DESCRIPTION
**Pull request by andrecolin**
This change is [Reviewable](https://reviewable.io/reviews/att-comdev/dockerfiles/68)

### CODE REVIEW

**Comment by aric49** [comment]
Tested locally, LGTM!

File: slackbridge/Dockerfile [hunk 1]
```diff
@@ -1,39 +1,30 @@
-FROM ubuntu:wily
+FROM ubuntu:16.04
 
 ENV DEBIAN_FRONTEND noninteractive
 
-ENV irc_server "irc.server.net"
```
Comment (v1k0d3n) [inline slackbridge/Dockerfile:5]: this is good too. using an env-file is fine.

File: slackbridge/Dockerfile [hunk 2]
```diff
@@ -1,39 +1,30 @@
-FROM ubuntu:wily
+FROM ubuntu:16.04
```
Comment (v1k0d3n) [inline slackbridge/Dockerfile:1]: yes! 16.04 :)

**Review by v1k0d3n — APPROVED** [review]
this all looks good. thanks for the updates @andrecolin

**Pull request merged**